src/service/plugins/mousepad.js

```diff
@@ -192,7 +192,7 @@ var Plugin = GObject.registerClass({
                 // Special key (eg. non-printable ASCII)
                 } else if (input.specialKey && KeyMap.has(input.specialKey)) {
                     keysym = KeyMap.get(input.specialKey);
-                    this._input.pressKeys(keysym, modifiers);
+                    this._input.pressreleaseKeysym(keysym);
                     this._sendEcho(input);
```

Comment:
@andyholmes @bevsxyz

Ooh, wait, what about the `modifiers`? They're not part of the `keysym` value (or, I don't see how they would be), so  it seems like they're getting lost here.

Reply:
The input's `._pressKeysym()` (as opposed to `.pressReleaseKeysym()` takes a `modifiers` argument, and that's what `.pressKey()` (not `.pressKeys()`) calls internally when passed an integer first argument (assumed to be a `keysym`):

https://github.com/GSConnect/gnome-shell-extension-gsconnect/blob/281a9c9c4cf156374bf651f0956ecfc04ba08771/src/service/components/atspi.js#L290-L294

Reply:
Ah, I was wrong after all. It looks like that's what `Input.pressKeys()` already does:

https://github.com/GSConnect/gnome-shell-extension-gsconnect/blob/281a9c9c4cf156374bf651f0956ecfc04ba08771/src/service/components/input.js#L608-L621

I guess this is really just a bug that was already fixed, but not included in a release :shrug:

Reply:
@andyholmes 
Sounds like it. On that assumption, I'm going to close this one. Sounds like we can probably close #1409 as well, tho I suppose there's no harm in waiting until v54 is out there and we can confirm that this is already fixed already.